pep-0678.rst

```diff
@@ -319,8 +319,8 @@ proposed ``__notes__`` semantics, but this would be rarely and inconsistently
 applicable.
 
 
-Store notes in ``ExceptionGroup``\ s
-------------------------------------
+Store notes in ``ExceptionGroup``\ s, but not other ``Exception``\ s
```

Comment:
Besides being long and unwieldly, this heading is rather confusing to me who's been passively following this PEP but has not delved into the nitty gritty details of its content. as a non-expert on its content. Specifically, is it saying that that other ``Exceptions``s do not have ``notes`` stored in them (which I *think* is the case, from having read the SC issue)? Or that ``Exceptions`` are not stored in ``ExceptionGroup``s? I'm assuming the former, but the present syntax is ambiguous and could confuse someone not reading it carefully multiple times, or not particularly familiar with the new ``ExceptionGroups`` and ``notes`` constructs.

Reply:
...yeah, that section hasn't aged well; I'll rewrite entirely to clarify.